src/views/ShareCredentialSelect/ShareCredentialSelect.tsx

```diff
@@ -139,15 +144,16 @@ export function ShareCredentialSelect({
       <section
         className={styles.allCredentials}
         ref={ref}
-        hidden={!hasSome.current}
+        hidden={noMatchingCredentials}
```

Comment:
Is there a reason we use `hidden` here instead of just not including this section at all? This could be a good moment to change that.